templates/galaxy/config/aarnet_object_store_conf.xml.j2

```diff
@@ -1,34 +1,44 @@
 <?xml version="1.0"?>
-<object_store type="hierarchical">
+<object_store type="distributed" id="primary" order="0">
     <backends>
-        <backend id="aarnetNFS7" type="disk" order="0">
+        <!-- New folder with storage by UUID on the 50T volume. Initially set weight=0.  b
+        one that will be written to if all of the others have weight 0  -->
+        <backend id="data009" weight="0" store_by="uuid">
+            <files_dir path="/mnt/user-data-6/data009" />
+        </backend>
+        <!-- New folder with storage by UUID on the 150T volume, weight=1 means it is the only
+        one that will be written to if all of the others have weight=0 -->
+        <backend id="data008" weight="1" store_by="uuid">
+            <files_dir path="/mnt/data008" />
+        </backend>
+        <backend id="aarnetNFS7" type="disk" order="0" store_by="id">
```

Comment:
So the intention is that /mnt/data008 is the one written to? Ie on the 150T volume. The comment on data009 is a bit confusing. "... b <newline> one that will be written to if all others have weight 0..." is this something to do with user-data-6/6b ?

Reply:
the second line of comment on 009 has been pasted in error

Reply:
thanks. that makes more sense.